packages/firestore/src/util/obj.ts

```diff
@@ -58,6 +58,12 @@ export function forEachNumber<V>(
   }
 }
 
+export function values<V>(obj: Dict<V>): V[] {
```

Comment:
This strikes me as very similar to https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_objects/Object/values.

Reply:
Seems unavailable on some of our platforms :(